app/views/posts/index.html.erb

```diff
@@ -2,8 +2,12 @@
     <div class="mb-5">
         <%= render partial: "partials/userCard", locals: { user: @user } %>
     </div>
+    <div class="w-full flex justify-end ">
```

Comment:
### Optional
- [ ] It would be better if we can click on the post to be redirected to the specific post, I was able to achieve the functionality manually by entering the routes in the browser